src/deploy-actions.js

```diff
@@ -34,9 +34,11 @@ const filterableItems = ['apis', 'triggers', 'rules', 'dependencies', ...package
  * @param {Array} [deployConfig.filterEntities.apis] filter list of apis to deploy, e.g. ['name1', ..]
  * @param {Array} [deployConfig.filterEntities.dependencies] filter list of package dependencies to deploy, e.g. ['name1', ..]
  * @param {object} [logFunc] custom logger function
+ * @param {object} [options={}] options object to store params
+ * @param {boolean} [options.actionCode] param: if false, skips action code deployment
  * @returns {Promise<object>} deployedEntities
  */
-async function deployActions (config, deployConfig = {}, logFunc) {
+async function deployActions (config, deployConfig = {}, logFunc, options) {
```

Comment:
For the tests:
1. Change `deployActions` signature to:
`async function deployActions (config, deployConfig = {}, log = console.log, options = {}) {`
2. Remove line https://github.com/adobe/aio-lib-runtime/blob/0e6ce80979816bd2ed1712ceed849702f164d38c/src/deploy-actions.js#L43

This should at least have defaults for all the parameters, but it will not complete coverage.